packages/backend/src/services/UserService.ts

```diff
@@ -1318,8 +1318,12 @@ export class UserService extends BaseService {
         ].filter(Boolean) as OpenIdIdentityIssuerType[];
```

Comment:
Just noticed that `GENERIC_OIDC` is not accounted for in this array.

Reply:
good catch, I added it here